orbit/pkg/update/runner.go

```diff
@@ -43,13 +45,16 @@ func NewRunner(client *Updater, opt RunnerOptions) (*Runner, error) {
 	// This is an optimization to not compute the hash of the local files every opt.CheckInterval
 	// (knowing that they are not expected to change during the execution of the runner).
 	localHashes := make(map[string][]byte)
-	for target, channel := range opt.Targets {
-		meta, err := client.Lookup(target, channel)
+	for _, target := range opt.Targets {
+		meta, err := updater.Lookup(target)
```

Comment:
So while the Runner is the thing triggering the update check, the Updater is responsible for keeping track of what channel we might want for what thing. Correct? That's why the Runner only cares about the targets now.

Reply:
Correct, it was previously being set in the `Updater` and in the `Runner` again. 
Now we set the channels in the Updater and we tell the runner which targets we want to "run update checks" to.